Releases.md

```diff
@@ -6,6 +6,23 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at:
 https://github.com/denoland/deno_install
 
+### 1.7.2 / 2021.02.05
+
+- feat(lsp): add references code lens (#9316)
+- feat(lsp): add TS quick fix code actions (#9396)
```

Comment:
feat(lsp, unstable) ? (just to deflect inevitable complaints we're not sticking to semver?)

Reply:
Sure, I'll rework it